oak-jcr/src/main/java/org/apache/jackrabbit/oak/jcr/session/ItemImpl.java

```diff
@@ -17,6 +17,7 @@
 package org.apache.jackrabbit.oak.jcr.session;
 
 import static com.google.common.collect.Lists.newArrayListWithCapacity;
+import static java.lang.String.format;
```

Comment:
Nitpick, personal style preference: I was confused reading the code, wondering if the method call format() was part of any 3rd party library or some util class we have made. It took me some time until realizing that it was a static import. My personal preference is to avoid static imports unless it is idiomatic (like Unit tests assert methods), as I think they can make the code a bit less readable.

Reply:
Yes, I agree! I did this only because NodeImpl.java already had that before... and I think consistency with that existing code is more important than my reluctance to use static import :-)

Reply:
maybe remove it in NodeImpl then :-)

Reply:
Removing it in NodeImpl would result in a 225 lines diff :-) So probably easier to keep it.

Reply:
Oops :-)